packages/@aws-cdk/integ-runner/lib/utils.ts

```diff
@@ -41,6 +41,18 @@ export function chain(commands: string[]): string {
   return commands.filter(c => !!c).join(' && ');
 }
 
+/**
+ * Prepare command for execution by exec
+ */
+export function prepareForExec(command: string): string[] {
+  let chunks = command.split(/\s+/);
+  const cmd = chunks.shift();
+  if (cmd !== undefined) {
+    return [cmd, chunks.join(' ')];
```

Comment:
Should this be `return chunks` instead?

For example if I had 

```
hooks: {
  preDeploy: ['npx cdk deploy'],
}
```

I would want it to return `['npx', 'cdk', 'deploy']` right?

Reply:
@corymhall, thank you for the comment, for the command you mentioned it could be like that. But for example for the command `echo "preDeploy hook"` if we just split it by space it will result in `['echo', '"preDeploy', 'hook"'] `which seems to be not the correct input for `exec`.

Reply:
You could do something like this instead in order to keep quoted sections together.

```ts
let chunks = command.match(/(?:[^\s"]+|"[^"]*")+/g)
// ['echo', '"preDeploy hook"']
```

I think if you use arguments with spawnSync each argument to the command needs to be its own element in the list.

Reply:
Yes, make sense, I updated the solution. Thank you again for the review.